devdox/app/services/git_tokens_service.py

```
@@ -130,122 +128,73 @@ def mask_token(token: str) -> str:
-        
+
         encrypted_token = self.crypto_store.encrypt_for_user(token, user.encryption_salt)
-        
-        new_payload: GitLabelCreate = GitLabelCreate(
-            label=json_payload.label,
-            user_id=user_claims.sub,
-            git_hosting=json_payload.git_hosting,
-            token_value=json_payload.token_value,
+
+        fetcher, _ = retrieve_git_fetcher_or_die(
+	        store=self.git_manager, provider=json_payload.git_hosting, include_data_mapper=False
```

Flake8 (7.2.0)

[error] indentation contains mixed spaces and tabs

(E101)